docs/autogen_config.py

```diff
@@ -11,7 +11,36 @@
 options = join(here, 'source', 'config', 'options')
 generated = join(options, 'config-generated.txt')
 
-from ipython_genutils.text import indent, dedent
+import textwrap
+indent = lambda text,n: textwrap.indent(text,n*' ')
+
+def dedent(text):
```

Comment:
Is this not just inspect.cleandoc?

Reply:
looks like it, do you want to make a PR?

Reply:
done in #11461